gsk4/src/auto/mod.rs

```diff
@@ -19,6 +19,13 @@ mod gl_shader;
 pub use self::gl_shader::GLShader;
 pub use self::gl_shader::GLShaderBuilder;
 
+#[cfg(any(feature = "v4_2", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_2")))]
+mod ngl_renderer;
```

Comment:
Why is this public API at all?

Reply:
so that you can force a specific renderer I guess?

Reply:
Ok :)